driver.go

```diff
@@ -123,8 +159,13 @@ func (l *lvmDriver) Create(req volume.Request) volume.Response {
 
 	v := &vol{Name: req.Name, MountPoint: mp}
 	if isSnapshot {
+		source := l.volumes[snap]
 		v.Type = "Snapshot"
 		v.Source = snap
+		v.KeyFile = source.KeyFile
```

Comment:
I see a use case for backing up encrypted volumes via snapshot, even if the key is not available. Mount will now check that the key is present when the snapshot is used.